tools/ci-operator-prowgen/README.md

```diff
@@ -0,0 +1,140 @@
+# Prow job generator for ci-operator
+
+The purpose of this tool is to reduce an amount of boilerplate that component
+owners need to write when they use
+[ci-operator](https://github.com/openshift/ci-operator) to set up CI for their
+component. The generator is able to entirely generate the necessary Prow job
+configuration from the ci-operator configuration file.
+
+## Use
+
+To use the generator, you need to build it:
+
+```
+$ go build
```

Comment:
From root of repo?

```
go build tools/ci-operator-prowgen/...
```

Reply:
I thought it would be better for the snippets to match the README location but OK.